go/extractor/BUILD.bazel

```diff
@@ -4,6 +4,7 @@ load("@rules_pkg//pkg:mappings.bzl", "pkg_files")
 # gazelle:prefix github.com/github/codeql-go/extractor
 # gazelle:map_kind go_binary codeql_go_binary //go:rules.bzl
 
+# following target is kept up to date by `bazel run //go:gen`, do not edit directly
```

Comment:
If I understood our previous conversation correctly, this file isn't treated the same as the other auto-generated `.bazel` files because there are bits here we _might_ want to edit by hand? If so, maybe this wording would be better:
```suggestion
# targets in this file can automatically be kept up-to-date by `bazel run //go:gen` and do not need to be edited directly 
```

Reply:
it's really just the target immediately under this comment that is updated. All the other targets are hand-written and will remain so. I'll think of a more specific wording.